Update example program in README.md.

mkovalchik opened the pull request:
Updated the example program in the basic usage section of the readme
to build and run using the current version of rust/rustless.

GitCop commented:
There were the following issues with your Pull Request

 * Commit: 0e0745b46279fb11bf246c475ed9895a4b29e35e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/rustless/rustless

---
This message was auto-generated by https://gitcop.com

mkovalchik commented:
Whoops, updated the changeset comment to match the gitcop suggestion and cleaned up a little .gitconfig messup I had where I was making commits as someone else.

s-panferov commented:
Thanks, @mkovalchik!
